Gopkg.toml

```diff
@@ -25,7 +25,7 @@
 
 [[constraint]]
   name = "github.com/Azure/go-autorest"
-  version = "12.3.0"
+  version = "13.0.0"
```

Comment:
I will take care of this

Reply:
> This change (which is good) will disable tracing by default. Can you update the README according? See the CHANGELOG from `go-autorest` for details.

I updated the readme accordingly. Since today is the last day of August, I just release this version in advance. If there is anything that should also be included in the README, we can include those on next minor version.